dcicutils/deployment_utils.py

```diff
@@ -590,6 +595,9 @@ def build_ini_stream_from_template(cls, template_file_name, init_file_stream, *,
                                        if cls.APP_ORCHESTRATED
                                        else cls.LEGACY_FOURSIGHT_BUCKET_PREFIX))
         s3_bucket_env = s3_bucket_env or os.environ.get("ENCODED_S3_BUCKET_ENV", get_bucket_env(bs_env))
+        s3_encrypt_key_id = (s3_encrypt_key_id
+                             or os.environ.get("ENCODED_S3_ENCRYPT_KEY_ID")
+                             or "MISSING_ENCODED_S3_ENCRYPT_KEY_ID")
```

Comment:
I think this value needs to be `None`, otherwise code later on will look for a KMS key under `MISSING_ENCODED_S3_ENCRYPT_KEY_ID`?

Reply:
OK, I'll do that.

Reply:
Changed in later work to the empty string but an OK change for this PR.